tests/internal/scripts/ccuda/cuda_synchronize.py

```diff
@@ -0,0 +1,5 @@
+from pyccel import cuda
+
+if __name__ == '__main__':
+    cuda.synchronize()
+    cuda.synchronize()
```

Comment:
why are you  using 2 calls to `cuda.synchronize()`

Reply:
No reason in particular, its just they are supposed to be used by pair when calling a kernel launch i thought that maybe leaving this way wasn't a problem.

Reply:
Removed in [2a851ed](https://github.com/pyccel/pyccel/pull/1235/commits/2a851edd0ce5fd41a0251345c43b7ab0b462fd1c).